packages/flame/example/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
-name: position_component
-description: A new Flutter project.
+name: flame_example
```

Comment:
This one is also removed in the other PR, but better to do the removal in this PR since my PR wasn't related.

Reply:
Do you mean -- delete this sub-project entirely?

Reply:
Aah no, just rename, badly worded from my side.

Reply:
Ok, so no need to change anything, right? (since it's already renamed)

Reply:
Indeed, I just have to remove the change from my PR